x/agent.py

```diff
@@ -176,9 +184,10 @@ def play(self, env, epoch=1, batch_size=1, visualize=None, verbose=1):
             if verbose == 1:
                 progbar.add(1, values=[("loss", loss), ("rewards", rewards)])
 
+
         if visualize:
             print("Making gif!")
             frames = np.repeat(frames, 3, axis=-1)
-            make_gif(frames[:-visualize['n_frames']],
-                     filepath=visualize['filepath'], gray=visualize['gray'])
+            make_gif(frames[:visualize['n_frames']],
+                     filepath=visualize['filepath'], gray=visualize['gray'], interpolation='none')
```

Comment:
I'm getting this error when running the catcher experiment TypeError: make_gif() got an unexpected keyword argument 'interpolation'